sw/device/tests/sensor_ctrl_alerts.c

```diff
@@ -182,6 +169,26 @@ bool test_main(void) {
                                              /*ecc_en*/ false,
                                              /*he_en*/ false);
 
+  // If all bits are set, then this must be the first iteration
+  uint32_t max = (1 << SENSOR_CTRL_PARAM_NUM_ALERT_EVENTS) - 1;
+  uint32_t new_val;
```

Comment:
These two variables seem only used on the first iteration, so move their declaration inside the if below.

Reply:
updated